cime/src/drivers/mct/main/prep_ocn_mod.F90

```diff
@@ -1066,6 +1110,8 @@ subroutine prep_ocn_merge( flux_epbalfact, a2x_o, i2x_o, r2x_o, w2x_o, g2x_o, xa
                x2o_o%rAttr(index_x2o_Faxa_snow_HDO ,n)
        end if
     end do
+    !Experimental: call subshelf boundary flux code here.  Note this is called on atmospheric timestep.
```

Comment:
@JeremyFyke , can you elaborate on this comment?  What is experimental about this?  Why is it on the atmospheric timestep?  Would it ideally be called on the ocean coupling timestep?

Reply:
I *think* perhaps it could be called on the ocean coupling timestep, equivalently.  I believe 'experimental' refers to that of all this development, the impact of coupler timesteps was/is the least clear to me.  I think, for example, that my implementation may be doing redundant accumulation/averaging steps in the coupler.  Also, I didn't test different coupler timestep schemes (e.g. as explored by @proteanplanet)  @rljacob - I think it would be really worthwhile for you to take a look at the implementation here to see if you see any errors.